test/Microsoft.NET.Build.Tests/GivenThatWeWantToGenerateADepsFileForATool.cs

```diff
@@ -1,258 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System.Runtime.CompilerServices;
-
-using Microsoft.DotNet.Cli.Utils;
-
-using NuGet.Packaging;
-using NuGet.ProjectModel;
-
-namespace Microsoft.NET.Build.Tests
-{
-    public class GivenThatWeWantToGenerateADepsFileForATool : SdkTest
-    {
-        public GivenThatWeWantToGenerateADepsFileForATool(ITestOutputHelper log) : base(log)
-        {
-        }
-
-        //  Disabled on full Framework MSBuild due to https://github.com/dotnet/sdk/issues/1293
-        [CoreMSBuildOnlyFact]
-        public void It_creates_a_deps_file_for_the_tool_and_the_tool_runs()
```

Comment:
surprised that these tests weren't able to be kept around. could we set LatestMajor as an additional property perhas?